zulipterminal/ui_tools/boxes.py

```diff
@@ -1734,6 +1734,14 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     recipient_user_ids=self.recipient_ids,
                 )
             elif self.message["type"] == "stream":
+                is_unauthorised_warning = self.model.is_unauthorised_to_post_in_stream(
+                    self.message["stream_id"]
+                )
+                if is_unauthorised_warning is not None:
+                    print(is_unauthorised_warning)
```

Comment:
Maybe we should lint for print statements :)

Reply:
Oh sorry. I completely forgot to remove it before pushing the commit.